client/my-sites/term-tree-selector/README.md

```diff
@@ -0,0 +1,104 @@
+TermTreeSelector
```

Comment:
@mtias Any thoughts on having this component in `my-sites`, given discussions around app components? (In any case, I think it'd be fine to move after merging this one)